### ISSUE TITLE
Release a executable file

### ISSUE THREAD

**Issue by anandakrishnas** [lmdpua/A2168_Laptop_Battery_Flasher_on_CP2112#1]
Hey nice work, I dont know any idea about compile this. So can you make an executable file as release. 
Thanks in advance

**Comment by lmdpua** [comment]
> Hey nice work, I dont know any idea about compile this. So can you make an executable file as release. Thanks in advance

Hello. I compiled the project and added it to the releases section. 
Here is the direct [link](https://github.com/lmdpua/A2168_Laptop_Battery_Flasher_on_CP2112/files/11929660/A2168_CP2112_FLASHER_V0.1_RELEASE.zip)

**Comment by anandakrishnas** [comment]
yo man, nice work, Thanks for your support, Can I add this link to my youtube video, I will add credits.

**Comment by lmdpua** [comment]
You are welcome. Yes of course you can add

**Comment by bennecke1988-hub** [comment]
Hello, Friend! Thanks for the nice job! I've just finished the reset of my A2168 for HP Envy. I would be glad to share with All my practice and experience. For the future it would be nice to add automatic CRC calculations after the dump modification to provide safe download. Now all manual EPROM modifications can lead to PF flags in case of CRC mismatches. But finally I found the "way" to this battery with your program.
